### PR TITLE
LPS-133409 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133409

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Add a site via JSON") {
+			JSONGroup.addGroup(groupName = "Test Site Name");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:17]: Unneeded step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:28]: Unneeded step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	}
+
+	@description = "This is a test for LPS-121200. Can create a fragment collection in site named with XSS without receive an alert message."
+	@priority = "4"
+	@uitest
+	test CreateFragmentCollectionInSiteNamedWithXSS {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:34]: AddFragmentCollectionWhenSiteNameWithXSS

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		var siteName = '''<script>alert(123)</script>''';
+
+		task ("Add a site") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:37]: Add a site with XSS name

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+			ProductMenu.gotoPortlet(
+				category = "Design",
+				portlet = "Fragments");
+
+			FragmentsAdmin.addCollection(collectionName = "Collection Name");
+		}
+
+		task ("View the alert not shown") {
+			AssertAlertNotPresent();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:53]: Redundant step

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update the case according to comments? Thanks

**Comment by ozysouza** [comment]
Hi @Tim-Cao PR updated.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+		task ("Delete the site") {
+			JSONGroup.deleteGroupByName(groupName = "${siteName}");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:50]: Unneeded step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+	@description = "This is a test for LPS-121200. Can create a fragment collection in site named with XSS without receive an alert message."
+	@priority = "4"
+	@uitest
+	test AddFragmentCollectionWhenSiteNameWithXSS {
+		var siteName = '''<script>alert(123)</script>''';
+
+		task ("Add a site with XSS name") {
+			JSONGroup.addGroup(groupName = "${siteName}");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithXSS.testcase:32]: Use UI steps

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes in https://github.com/brianchandotcom/liferay-portal/compare/795cfe23f757d58c7aa9136e431ab77a8e310c38...036511217cfa48dd85bdca14b163123392a5a496

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**